### PR TITLE
[62] Create and style the customer sign up page with CSS

### PR DESCRIPTION
**Pull request by ToluEnny24**
I have created a sign up page and styled with css
Greenprint

### CODE REVIEW

**Comment by adaugochi** [comment]
Well done @Tocyn 

But, I still feel you can do better. Kindly rework your footer section and add a picture of the way it looks on mobile

**Comment by ToluEnny24** [comment]
i have edited my footer and will be adding the mobile and tablet views.
mobile view1
mobile view2
tab view
web view

**Comment by ToluAkin** [comment]
@ToluEnny24 The footer on small screen is not responsive. Can you take time to redesign it again and re-upload another picture? Well on your attempts so far.

**Comment by ToluEnny24** [comment]
This is the new look of my mobile view, I hope the footer is better right. Thanks
mobile view3.

**Comment by ToluAkin** [comment]
@ToluEnny24 Have everything in the footer be centered on small screen.

**Comment by ToluEnny24** [comment]
This is what the footer looks like now.
mobile view1
mobile view2

**Comment by adaugochi** [comment]
The form input labels should not be centralized

**Comment by ToluEnny24** [comment]
mobile view3

File: submissions/fabunmiTolulope/html-forms/contact-form.html [hunk 1]
```diff
@@ -13,8 +13,8 @@ <h1>Welcome to Greenprint Integrated Services</h1>
         <ul>
                 <li><a href="#"> Home</a></li>
                 <li><a href="#">Blog</a> </li>
-                <li><a href="../HTML/Productpage.html">Productpage</a> </li>
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/html-forms/contact-form.html:16]: Why are you removing the links?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/html-forms/contact-form.html (hunk 1 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/html-forms/contact-form.html:16]: @adaugochi the links I removed was only for the images,  I had them in a folder called images in the css folder and didn't want you to complain about it..
Then I don't understand that line 16 you highlighted.. thanks

Comment (ToluEnny24) [reply submissions/fabunmiTolulope/html-forms/contact-form.html:16]: oooh sorry my bad, I felt we were not told to link it together that was why I removed the links.

Comment (ToluEnny24) [reply submissions/fabunmiTolulope/html-forms/contact-form.html:16]: @adaugochi i have added the navigation links.

File: submissions/fabunmiTolulope/css/style.css [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+.footer1 p{
+	cursor: pointer;
+	text-align: center;
+    padding: 10px;
+}
+
+/**Media queries for mobile**/
+
+@media screen and (max-width: 450px) {
+	*body{
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/css/style.css:136]: What is this targeting? I don't understand `*body`

File: submissions/fabunmiTolulope/css/style.css [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+	text-align: center;
+    padding: 10px;
+    font-size: smaller;
+	}
+}
+
+/**Media queries for web*/
+
+@media screen and (min-width: 600px) {
+*body{
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/css/style.css:272]: same here too

File: submissions/fabunmiTolulope/html-forms/contact-form.html [hunk 4]
```diff
@@ -13,8 +13,8 @@ <h1>Welcome to Greenprint Integrated Services</h1>
...
-                <li><a href="../html-forms/contact-form.html">Contact Us</a> </li>
+                <li><a href="#">Productpage</a> </li>
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/html-forms/contact-form.html:16]: Revert the change to how it was before. Add the links back

File: submissions/fabunmiTolulope/html-forms/contact-form.html [hunk 5]
```diff
@@ -13,8 +13,8 @@ <h1>Welcome to Greenprint Integrated Services</h1>
...
+                <li><a href="#">Contact Us</a> </li>
```
Comment (adaugochi) [inline submissions/fabunmiTolulope/html-forms/contact-form.html:17]: Same here too

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/fabunmiTolulope/html-forms/contact-form.html (hunk 4 above)
Comment (ToluEnny24) [reply submissions/fabunmiTolulope/html-forms/contact-form.html:16]: I have added the navigation links

Comment (ToluEnny24) [reply submissions/fabunmiTolulope/html-forms/contact-form.html:16]: Screenshot_20201030-121839_Samsung Internet

**Comment by ToluEnny24** [comment]
i have added the navigation links and made changes to the stylesheett

**Review by adaugochi — CHANGES_REQUESTED** [review]
Click [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/776/files#diff-951cef376681e5f41b34e144ec7076fb4c844b8c96034a2104786bb3991ee292) to view your PR. you haven't added the navigation links you removed

**Review by adaugochi — APPROVED** [review]
